src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  PaginationWrap,
+  PaginationButtonsWrap,
+  PaginationItem,
+  PaginationDots,
+  PaginationButton,
+  PaginationNavigation,
+} from './styles';
+import PrevArrow24 from '../Icons/PrevArrow24';
+import NextArrow24 from '../Icons/NextArrow24';
+
+const Pagination = ({ currentPage, total, onPageChange }) => {
+  const [buttons, setButtons] = useState([]);
+  const numberOfPages = [];
+
+  for (let i = 1; i <= total; i++) { //eslint-disable-line
+    numberOfPages.push(i);
+  }
+
+  useEffect(() => {
+    let paginationArray;
```

Comment:
Set this variable as empty array by default. Also, maybe name it just `pagination`, wdyt?